backend/api/schema.py

```diff
@@ -0,0 +1,186 @@
+import graphene
+from graphene_django import DjangoObjectType
+from .models import *
+
+
+class UserType(DjangoObjectType):
+    class Meta:
+        model = User
+        fields = ('id', 'username', 'email', 'first_name', 'last_name',
+                  'is_staff', 'is_superuser', 'is_active', 'date_joined', "publickey")
+
+
+class WalletType(DjangoObjectType):
+    class Meta:
+        model = Wallet
+
+
+class TransactionType(DjangoObjectType):
+    class Meta:
+        model = Transaction
+
+
+class CurrencyType(DjangoObjectType):
+    class Meta:
+        model = Currency
+
+
+class Query(graphene.ObjectType):
+    all_users = graphene.List(UserType, description="List of all users")
+    user_by_id = graphene.Field(
+        UserType, id=graphene.Int(), description="Get user by id")
+    user_by_username = graphene.Field(
+        UserType, username=graphene.String(), description="Get user by username")
+
+    all_wallets = graphene.List(WalletType, description="List of all wallets")
+    wallet_by_id = graphene.Field(
+        WalletType, id=graphene.Int(), description="Get wallet by id")
+    wallets_by_user = graphene.List(
+        WalletType, user=graphene.String(), description="List of all wallets of a user")
+    wallets_by_currency = graphene.List(WalletType, currency=graphene.String(
```

Comment:
`.String()`'s parantheses are on different lines. If you are using a formatter, you can put a line break before `currency` to prevent it from hitting the line limit and avoid this.